### PR TITLE
Update release notes for 1.19.0

### PR DESCRIPTION
**Pull request by tbrisker**
Also make sure the script only lists each release once

### CODE REVIEW

File: _includes/manuals/1.19/1.2_release_notes.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ Previously, the cloud-init integration for oVirt supported only a sub-set of clo
 * Pass full_name attribute for settings in API ([#24025](https://projects.theforeman.org/issues/24025))
 * Inconsistant documentation in --order option in scoped_search for katello and foreman ([#23769](https://projects.theforeman.org/issues/23769))
 * Could not able to see filters of any role with org_admin user ([#21778](https://projects.theforeman.org/issues/21778))
+* Foreman (1.18?) slow API call against fact_values endpoint ([#24712](https://projects.theforeman.org/issues/24712))
```
Comment (ekohl) [inline _includes/manuals/1.19/1.2_release_notes.md]: The `1.18?` doesn't sound very confident

File: _includes/manuals/1.19/1.2_release_notes.md [hunk 2]
```diff
@@ -109,11 +149,18 @@ Previously, the cloud-init integration for oVirt supported only a sub-set of clo
 #### Importers
 * Fact parser flag keep_subnet to prevent subnet override during fact upload ([#23768](https://projects.theforeman.org/issues/23768))
 
+#### Installer
+* foreman installer fails ([#24628](https://projects.theforeman.org/issues/24628))
```
Comment (ekohl) [inline _includes/manuals/1.19/1.2_release_notes.md]: I've updated this to be a bit more descriptive

File: _includes/manuals/1.19/1.2_release_notes.md (hunk 1 above)
Comment (tbrisker) [reply _includes/manuals/1.19/1.2_release_notes.md]: removed the question mark

File: _includes/manuals/1.19/1.2_release_notes.md (hunk 2 above)
Comment (tbrisker) [reply _includes/manuals/1.19/1.2_release_notes.md]: updated accordingly

**Comment by mmoll** [comment]
merged, thanks @tbrisker!

**Pull request merged**